src/randoop/main/GenInputsAbstract.java

```diff
@@ -349,53 +305,61 @@ public GenInputsAbstract(String command, String pitch,
   public static boolean no_regression_assertions = false;
 
   /**
-   * The possible values of the regression_assertions_about_exceptions command-line argument.
-   * @see #regression_assertions_about_exceptions
+   * The possible values for exception allocation command-line arguments.
    */
-  public static enum ExceptionAssertionsMode {
-    /** Include no exceptional tests in the regression test suite */
-    NONE,
-    /** Include tests that throw checked exceptions in the regression test suite */
-    CHECKED,
-    /** Include every test in the regression test suite */
-    ALL;
+  public static enum BehaviorType {
+    /** Occurrence of exception should be considered an error */
+    ERROR,
+    /** Occurrence of exception should be considered invalid */
+    INVALID,
+    /** Occurrence of exception should be considered expected behavior */
+    EXPECTED
   }
-
+  
   /**
-   * If a test throws an exception, should it be included in the regression test suite?
+   * Indicates whether a checked exception should be considered as error,
+   * expected, or invalid behavior. Determines how the exception is handled
```

Comment:
"Determines how ..." feels like an implementation detail that I would omit.  This determines the classification of the test.  (Then it's the classfication of the test, not this, that determines how the exception is handled.)
Also, the text has become more obscure.  How can an exception be an error? The previous text was explicit:  it's the test that is being classified, not the exception, and what's relevant is that the test threw an exception.  So, please do improve the documentation, but don't take away from it.  I feel that you have changed this user-visible documentation to be terser and maybe more comprehensible to you, but that it will be confusing to users because it is less explicit and assumes they know much more about the internals of Randoop than they have any right to know.